### DISCUSSION TITLE
Firmware Update to CC2652R (ZZH) 20211217 with older Zigbee2mqtt version (1.17.1)

### DISCUSSION THREAD

**Discussion by Anjerlaan** [Koenkk/zigbee2mqtt#10861] [General]
Hi @Koenkk, 

**Hardware/Software:**
Raspberry Pi 3B+
Raspbian Buster
Domoticz 2021.1 (build 13490)
Python Version: 3.7.3
ZZH CC2652R_coordinator_20210120
Zigbee2mqtt 1.17.1

I haven't updated Zigbee2mqtt and the Domoticz zigbee2mqtt plugin for a while.
This is because the domoticz-zigbee2mqtt-plugin is still in a beta stage.
To avoid problems with the system during updating, I have 2 questions:

1. If I update ZZH firmware to 20211217 does it still run version Zigbee2mqtt 1.17.1?
2. If I create an IMAGE of the RPi's SD-Card to another SD-Card and mount it in the Pi, does zigbee2mqtt run properly, or do I still need to change/modify the Network and/or Pan-ID?

Greetzzz,

Gerben

**Comment by Koenkk** [comment]
> If I update ZZH firmware to 20211217 does it still run version Zigbee2mqtt 1.17.1?

should work (haven't tested it, but don't see why not)

> If I create an IMAGE of the RPi's SD-Card to another SD-Card and mount it in the Pi, does zigbee2mqtt run properly, or do I still need to change/modify the Network and/or Pan-ID?

That should also work (no repairing needed)

**Reply by Anjerlaan** [reply]
Thanx!
